torch/csrc/jit/export.cpp

```diff
@@ -676,13 +619,13 @@ void ModuleEncoder::EncodeMethod(
   EncodeBlock(attr_proto->mutable_g(), method.graph()->block(), {});
 }
 
-void ModuleEncoder::EncodeTensor(
+void MethodEncoder::EncodeTensor(
```

Comment:
@zdevito @li-roy - do we actually have Tensor block that might appear inside attributes of nodes of IR? My understanding is that it's not. So we could just disable it and remove this logic

Reply:
so is the plan to address the "arbitrary tensor storage" in a separate diff?

Reply:
Yes, constants contain tensor attributes.